__tests__/e2e/scripts/setup.sh

```diff
@@ -1,36 +1,13 @@
 #!/usr/bin/env bash
 set -e
 
-# variables
+# Update elastic-package
+go get github.com/elastic/elastic-package
 
-# ensure Docker is running
-docker ps &> /dev/null
-if [ $? -ne 0 ]; then
-    echo "⚠️  Please start Docker"
-    exit 1
-fi
+eval "$(elastic-package stack shellinit)"
 
-if [ -z "${JENKINS_URL}" ]; then
-  # formatting
-  bold=$(tput bold)
-  normal=$(tput sgr0)
-fi
+# Take the stafck down
+elastic-package stack down
 
-# paths
-E2E_DIR="./"
-TMP_DIR="tmp"
-
-#
-# Create tmp folder
-##################################################
-echo "" # newline
-echo "${bold}Temporary folder${normal}"
-echo "Temporary files will be stored in: ${E2E_DIR}${TMP_DIR}"
-mkdir -p ${TMP_DIR}
-
-
-echo "" # newline
-echo "${bold}Starting elasticsearch , kibana and synthetics docker${normal}"
-echo "" # newline
-
-STACK_VERSION=8.0.0-SNAPSHOT docker-compose --file docker-compose.yml up --remove-orphans > ${TMP_DIR}/docker-logs.log 2>&1 &
+# start elastic-package
+elastic-package stack up -d -v
```

Comment:
We would need an environment variable provided in the container to pass through the version we'd like to test.

Reply:
Do you need help from the robots team on this?